server/node.go

```diff
@@ -170,6 +170,12 @@ func NewNode(ctx storage.StoreContext) *Node {
 	}
 }
 
+// context returns a base context for Node-level use. It is shared with
+// the local sender and contains NodeID and ClusterID (once known).
+func (n *Node) context() context.Context {
```

Comment:
Where is Node.context used? It seems very strange to have Node and LocalSender share a context since they are at different layers of the stack, and some of the contextual information we want (specifically the client IP) will be added by the node and passed down through the senders. I think LocalSender.Send needs to accept a Context so that higher-level information can be passed through it.